### ISSUE TITLE
bad sanitization of strings inserted in libRocket document

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#1494]
Reported by ZdrytchX on chat, his nickname produces RML parsing error:

[https://cdn.discordapp.com/attachments/353835068139241483/885749548029472828/unknown.png](https://cdn.discordapp.com/attachments/353835068139241483/885749548029472828/unknown.png)

His nickname contains `>` and `<` characters: `>KoR<Frisqy.Virus`

The error is:

```
Warn: Closing tag 'span' mismatched on :2 was expecting 'frisqy.virus"'.
```

**Comment by bmorel** [comment]
Let me place the screenshot here so that it does not get lost when link dies.
![unknown](https://user-images.githubusercontent.com/1316300/177559904-d06193a4-ac3d-4347-a3e8-38662a5f4351.png)

Ideally, this should be re-verified in 0.53 since it might be related to librocket, replaced with rmlui.

**Comment by slipher** [comment]
I think I fixed this one.

**Comment by bmorel** [comment]
I can't reproduce this on illwieckz's nor on my servers... this was fixed before 0.52.1?

**Comment by slipher** [comment]
d2aaff72 is the one I was thinking of.

Whatever, let's close it.
